src/main/java/net/dv8tion/jda/internal/requests/Route.java

```diff
@@ -216,6 +217,18 @@
         public static final Route GET_PERM_OVERRIDE =    new Route(GET,    "channels/{channel_id}/permissions/{permoverride_id}");
         public static final Route FOLLOW_CHANNEL =       new Route(POST,   "channels/{channel_id}/followers");
 
+        public static final Route START_THREAD_WITH_MESSAGE =               new Route(POST, "channels/{channel_id}/messages/{message_id}/threads");
+        public static final Route START_THREAD_WITHOUT_MESSAGE =            new Route(POST, "channels/{channel_id}/threads");
```

Comment:
```suggestion
        public static final Route CREATE_THREAD_WITH_MESSAGE =               new Route(POST, "channels/{channel_id}/messages/{message_id}/threads");
        public static final Route CREATE_THREAD_WITHOUT_MESSAGE =            new Route(POST, "channels/{channel_id}/threads");
```

Reply:
Also, please align the route strings like the other routes have done

Reply:
[I followed the naming in the Discord docs](https://discord.com/developers/docs/resources/channel#start-thread-with-message)
Thus START_THREAD instead of CREATE_THREAD
If you prefer CREATE_THREAD that will be used instead

Reply:
statement wasn't about the naming, it was about the code style of the string routes

Reply:
We might want to go with `CREATE_` from a consistency perspective for inside of JDA.

Reply:
Understandable, will change